src/sentry/south_migrations/0229_drop_event_constraints.py

```diff
@@ -15,6 +15,8 @@ def forwards(self, orm):
             db.delete_foreign_key('sentry_eventmapping', 'project_id')
         except OperationalError:
             pass
+        except ValueError:
```

Comment:
Why not

```suggestion
        except OperationalError, ValueError:
```

above?

Reply:
That's not equivalent.

`except (OperationalError, ValueError):` is though.